### PR TITLE
Feature/organization auth (KHO-235, KHO-236)

### PR DESCRIPTION
**Pull request by ntietje1**
# Checklist

- [X] I have performed a self-review of my code
- [X] I have reached out to another developer to review my code
- [X] I acknowledge my responsibility to reinitialize the deployed DB if there are schema changes.
- [X] New and existing unit tests pass locally with my changes
- [X] All Lint and CI checks are passing

### CODE REVIEW

File: frontend/src/components/Button/index.tsx [hunk 1]
```diff
@@ -14,13 +14,26 @@ const ButtonWrapper: React.FC<IButtonProps> = ({
...
+        {children}
+      </a>
+    );
+  }
+
+  // For internal routing
+  if (!href.startsWith('http')) {
+    return <Link to={href}>{children}</Link>;
+  }
+
```
Comment (nandini-ghosh) [inline frontend/src/components/Button/index.tsx:34]: nice

File: frontend/src/pages/Organizations/Select/styles.css [hunk 2]
```diff
@@ -2,4 +2,13 @@
     &__buttonWrapper {
         padding: 10px;
     }
-}
\ No newline at end of file
+
+    &__link {
+        color: var(--primary-text);
```
Comment (nandini-ghosh) [inline frontend/src/pages/Organizations/Select/styles.css:7]: Eventually I will create global styles for links and remove this but good for now

**Review by nandini-ghosh — APPROVED** [review]
Looks neat 🥇

File: frontend/src/pages/Organizations/Select/index.tsx [hunk 3]
```diff
@@ -26,8 +28,12 @@ const OrganizationSelection: React.FC = () => {
         if (data.orgs_without_app) {
           setOrgsWithoutApp(data.orgs_without_app);
         }
-      } catch (_) {
-        // do nothing
+
+        const callbackData = await getCallbackURL();
+        setConsentUrl(callbackData.consent_url);
+        setError(null);
+      } catch (e) {
```
Comment (alexangione419) [inline frontend/src/pages/Organizations/Select/index.tsx:35]: this e can be an "_" to get rid of the lint warning

**Review by alexangione419 — APPROVED** [review]
classic blunder averted

**Review by kennybc — APPROVED** [review]

**Pull request merged**